### ISSUE TITLE
Loinc 13862-8:  "Dehydroepiandrosterone (DHEA) [Mass/volume] in Serum or Plasma --15 minutes post 250 ug corticotropin"

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#3768]
Suggest creating a new child term of HP:0500022 [Abnormal serum dehydroepiandrosterone level] for Loinc 13862-8 [Dehydroepiandrosterone (DHEA) [Mass/volume] in Serum or Plasma --15 minutes post 250 ug corticotropin]
New term label: Abnormal circulating dehydroepiandrosterone level
Increased circulating dehydroepiandrosterone level
Your biocurator ID for loinc2hpo (if desired): HPO:nvasilevsky

**Comment by pnrobinson** [comment]
"Abnormal circulating dehydroepiandrosterone level" is incorrect, since this measurement is checking a response to a stimulatory test with corticotropin. 
This is related to this term 
https://hpo.jax.org/app/browse/term/HP:0031074
However, I think that this is currently too complicated to code with a single HPO term since whether or not the response is abnormal will depend on context and other measurements. I would say we cannot annotate this LOINC code at the current time. @nicolevasilevsky @kingmanzhang

**Comment by kingmanzhang** [comment]
changed to unable to annotate
